source/docs/software/can-devices/third-party-devices.rst

```diff
@@ -109,3 +109,16 @@ PWF Sensors
 
     - API Documentation (`Java <https://www.playingwithfusion.com/frc/2020/javadoc/com/playingwithfusion/package-summary.html>`__, `C++ <https://www.playingwithfusion.com/frc/2020/cppdoc/html/annotated.html>`__)
     - `Technical Manual <https://www.playingwithfusion.com/include/getfile.php?fileid=7091>`__
+
+Redux Robotics
+--------------
+
+Redux Robotics currently offers the Canandcoder CAN-enabled magnetic encoder. The vendordep is necessary for CAN-based operation.
```

Comment:
The comment about vendordep seems a little odd because there's no link to the vendordep so it leaves the reader searching. It's also true of every other item on this page, but not stated. Recommend clarifying that it supports two modes of operation and library is only needed for CAN or removing, since that is covered in the linked manual.